fastlane/Fastfile

```diff
@@ -479,26 +480,9 @@ lane :xcmetrics do |options|
 
   UI.user_error!("See Firebase error above ☝️") unless firebase_error.to_s.empty?
 
-  if is_ci
-    pr_comment_required = !ENV['GITHUB_PR_NUM'].to_s.empty?
-    performance_benchmarks[current_branch] = actual_performance
-    File.write(xcmetrics_path, JSON.pretty_generate(performance_benchmarks))
-
-    Dir.chdir(File.dirname(xcmetrics_path)) do
-      if sh('git status -s', log: false).to_s.empty?
-        pr_comment_required = false
-        UI.important('No changes in performance benchmarks. Skipping commit and comment.')
-      else
-        sh('git add -A')
-        sh("git commit -m 'Update #{xcmetrics_path}: #{current_branch}'")
-        sh('git push')
-      end
```

Comment:
This is to eliminate the useless branch metrics in the baseline json file. Basically now performance metrics will keep only the benchmark, similar to size metrics.